src/cogs/utils.py

```diff
@@ -1,7 +1,10 @@
 import nextcord, os
 from nextcord.ext import commands
 from datetime import datetime, timedelta
-import zoneinfo
+try:
```

Comment:
Why was a try needed here?

Reply:
because it wouldn't work with pytz in python 3, so i put it in a try so it works no matter what.

Reply:
I'm not quite sure what you mean by that. This runs in production with Python3, as well as my workstation.